runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4441,23 +4415,7 @@ TR::CompilationInfoPerThreadRemote::cacheResolvedMethod(TR_ResolvedMethodKey key
    if (!useCaching)
       return;
 
-   TR::Compilation *comp = getCompilation();
-   if (comp->compileRelocatableCode() && comp->getOption(TR_UseSymbolValidationManager) && comp->getSymbolValidationManager()->inHeuristicRegion())
-      {
```

Comment:
I am guessing the fix talked about in this comment has been delivered.

Reply:
Yes.